frontend/src/metabase/admin/settings/components/widgets/EmbeddingOption/EmbeddingOption.styled.tsx

```diff
@@ -27,5 +27,5 @@ export const BoldExternalLink = styled(ExternalLink)`
   font-weight: bold;
   display: flex;
   align-items: center;
-  gap: calc(6 / 16 * 1rem);
+  gap: 0.5rem;
```

Comment:
This addresses the [comment from the previous PR.](https://github.com/metabase/metabase/pull/47189#discussion_r1740880367)

Reply:
Thanks!